pysmt/test/smtlib/parser_utils.py

```diff
@@ -194,3 +202,119 @@ def formulas_from_smtlib_test_set(logics=None):
     #
     (QF_UF, "small_set/QF_UF/test0.smt2.bz2", SAT),
 ]
+
+
+def omt_test_cases_from_smtlib_test_set(logics=None):
+    """
+    Returns a generator over the test-set of OMT-LIB test files.
+    The method accepts a set of logics; only the test cases
+    with a logic inside the logics set will be returned.
+    If logics is None, all the test cases will be returned.
+
+    Note: This resets the Environment at each call.
+    """
+    for (logic, fname, is_sat, expected_result) in OMTLIB_TEST_FILES:
+        if logics is not None and logic not in logics:
+            continue
+        reset_env()
+        smtfile = os.path.join(OMTLIB_DIR, fname)
+        parser = SmtLib20Parser()
```

Comment:
Why insisting on the 2.0 version of the parser? The basic one is more general...